IntroComponentWithSign-upForm/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "test-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "tailwindcss build tailwind.css -o style.css",
+    "watch": "tailwindcss build tailwind.css -o style.css -w"
+  },
+  "keywords": [],
+  "author": "",
```

Add your name and email here! Something like this:

```suggestion
  "author": "Sami Zad <sami.zad@gmail.com>",
```

Always a good practice.